packages/houdini-utils/src/util/blobify.ts

```diff
@@ -0,0 +1,51 @@
+import { PaintWorkletBase } from '../types';
+
+const stringify = (
+  name: string,
+  paintWorklet: typeof PaintWorkletBase | string,
+  className?: string
+) => {
+  let clsStr = '';
+  let clsName = className;
+
+  if (typeof paintWorklet === 'string') {
+    clsStr = paintWorklet;
+    if (!clsName) {
+      const nameFinder = /class\s([a-zA-Z0-9_]+)\s?\{/;
+      const match = clsStr.match(nameFinder);
+
+      if (!match) {
+        throw new Error('Could not find class name in string');
+      }
+
+      clsName = match[1];
+    }
+  } else {
+    clsStr = paintWorklet.toString();
+    if (!clsName) {
+      clsName = paintWorklet.name;
+    }
+  }
+
+  let str = clsStr;
+  str += '\n';
+  str += `registerPaint('${name}', ${clsName})`;
+
+  return str;
+};
+
+/**
+ * Creates a Blob from a paint worklet.
```

Comment:
Since we know this is likely to have issues with CSPs we should add a note about the limitations of this function.